schemas/_common-schema.json

```
@@ -73,6 +73,55 @@
         }
       }
     },
+    "usr": {
+      "type": "object",
+      "description": "User properties",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Identifier of the user"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the user"
+        },
+        "email": {
+          "type": "string",
+          "description": "Email of the user"
+        }
+      }
+    },
+    "connectivity": {
+      "type": "object",
+      "description": "Device connectivity properties",
+      "required": [
+        "status",
+        "interfaces"
+      ],
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "Status of the device connectivity",
+          "enum": ["connected", "not_connected", "maybe"]
+        },
+        "interfaces": {
+          "type": "array",
+          "description": "The list of available network interfaces",
+          "items": {
+            "type": "string",
+            "enum": ["wifi", "cellular", "internet", "loopback", "other"]
+          }
+        },
+        "cellular_technology": {
+          "type": "string",
+          "description": "The type of a radio technology used for cellular connection"
+        },
+        "cellular_carrier": {
```

How about `cellular_carrier_name`?
```suggestion
        "cellular_carrier_name": {
```
This will enable us scoping more carrier information in the future.